src/classes/CON_PreferredPhone_TEST.cls

```diff
@@ -445,27 +406,27 @@ private class CON_PreferredPhone_TEST {
     */
     @isTest 
     static void testUpdatePrefPhoneTOtherBulk() {
-        
-        List<contact> contactsToInsert = new List<contact>();
-        for(Integer i = 0; i < 100; i++) {
+        enablePreferredPhone();
+
+        List<Contact> contactsToInsert = new List<Contact>();
+        for (Integer i = 0; i < 100; i++) {
             Contact a = new Contact(FirstName='TestAccount' + i, LastName = 'JohnnyTest' + i );
```

Comment:
i know this was existing as well, but we should be consistent in our spacing in the parameters. I'm starting to like the more spread out version (I used to like no spaces but its growing on me) but we should pick one.